index.js

```diff
@@ -147,10 +147,27 @@ const plugin = fp(async function (app, opts) {
     schema = gateway.schema
     entityResolversFactory = gateway.entityResolversFactory
 
+    let gatewayInterval = null
+
+    if (typeof gateway.pollingInterval === 'number') {
```

Comment:
Should we inform the user (throwing an error or smth. like this) if their `pollingInterval` option is not a numeric value? Maybe that would help them to track down errors in case they've added the number as a string etc.

Reply:
Yes, It's a good point! :+1: